addons/web/static/src/webclient/navbar/navbar.scss

```diff
@@ -33,24 +36,27 @@
 
     .o_menu_sections {
         .o_nav_entry, .dropdown-toggle {
-            @extend %-main-navbar-entry-bg-hover !optional;
-        }
+            background: var(--NavBar-entry-backgroundColor, #{$o-navbar-background});
+            border: $border-width solid transparent;
```

Comment:
There's a trailing space before the property.
There are plugins on VS Code that can help you to find and remove these :)